cmd/rook/operator.go

```diff
@@ -54,14 +53,13 @@ func startOperator(cmd *cobra.Command, args []string) error {
 	logger.Infof("starting operator")
 	context := createContext()
 	context.ConfigDir = k8sutil.DataDir
-	context.KubeContext = kit.KubeContext{
-		MasterHost: host,
-		Clientset:  clientset,
-		RetryDelay: 6,
```

Comment:
Yes it does. https://github.com/kubernetes/apimachinery/blob/master/pkg/util/wait/wait.go#L220.